backend/dal/Repositories/ActivityRepository.cs

```diff
@@ -135,6 +144,16 @@ public bool Delete(long activityId)
                 this.Context.PimsActivityInstanceNotes.Remove(activityNote);
             }
 
+            foreach (var propAcqFile in instance.PimsActInstPropAcqFiles)
```

Comment:
nit, no need for tiny names

Reply:
Sure, I was following the naming convention of the entity but I don't mind lengthening these.